ui/assets/img/remove-btn.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 43.2 (39069) - http://www.bohemiancoding.com/sketch -->
+    <title>Remove btn</title>
+    <desc>Created with Sketch.</desc>
```

Comment:
Bah! This is bad on me. I need to figure out why Sketch is not exporting compressed svg for me. It's fine for now. I'll create a separate issue to deal with it later.